automation/project/Dependencies.scala

```diff
@@ -7,11 +7,13 @@ object Dependencies {
 
   val workbenchModelV   = "0.8-d97f551"
   val workbenchGoogleV  = "0.10-b95b2c1"
+  val serviceTestV = "0.1-c5a3162-SNAP"
```

Comment:
Should update to non-SNAP; I guess that is dependent on the workbench-libs PR being merged.